skyvern/forge/agent.py

```
@@ -893,6 +917,172 @@ async def fail_task(self, task: Task, step: Step | None, reason: str | None) ->
     @TraceManager.traced_async(
         ignore_inputs=["browser_state", "organization", "task_block", "cua_response", "llm_caller"]
     )
+    async def _handle_file_download_verification_fallback(
+        self,
+        *,
+        organization: Organization,
```

Ruff (0.14.5)

Unused method argument: `organization`

(ARG002)